yarn.lock

```diff
@@ -7296,7 +7296,8 @@ eslint-module-utils@^2.6.0:
     pkg-dir "^2.0.0"
 
 "eslint-plugin-ember-data@link:./packages/unpublished-eslint-rules":
-  version "3.21.0-alpha.0"
+  version "0.0.0"
```

Comment:
This seems like an odd change fwiw, has eslint-plugin-ember-data been version correctly? Wondering what caused this